### DISCUSSION TITLE
Chia 1.1.5 debug.log

### DISCUSSION THREAD

**Discussion by dannyboid** [Chia-Network/chia-blockchain#4882] [Support]
Hello

I have been mining for 3 weeks now on **Windows 10 Pro** I don't have a crazy setup or anything like that but its very upsetting for me and I want to cry. Because when I went to check `debug.log` yesterday, which I have never checked because I was so preoccupied on getting the perfect setup (stupid me) and it looks like I was never plotting :( 

So anyone who is brilliant in this field can you please help me out? I'm not the smartest when it comes to computers. 

Note: I also don't have a single successful log 😞 

Currently I have this warning and error  for a whole day now, nothing changed 
`WARNING  No keys present. Create keys with the UI, or with the 'chia keys' program`
`ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection None. Traceback (most recent call last):`

(maybe useful info) I also plotted on Mac M1 Laptop and later added the plots to my main machine because using the laptop was not sustainable for I needed  to use it.

Some of the other errors I had
`ERROR    Exception , exception Stack: Traceback (most recent call last):`
`Error with syncing: <class 'concurrent.futures.process.BrokenProcessPool'>Traceback (most recent call last):`
`ERROR    Error while trying to fetch from peer: Traceback (most recent call last)`
`ERROR    Loop exception in sync Was not able to add blocks 8067-8099. Traceback (most recent call last):`
`WARNING  Trying to ban localhost for 10, but will not ban`
A crap ton of these (below)
`WARNING  Wasn't able to add transaction with id     status 3 error: Err.DOUBLE_SPEND`
`WARNING  Plot G:\plot-k32-2021-05-05-20-08-a plot has a farmer public key that is not in the farmer's pk list.`

Stuff I use to farm
Windows 10 Pro
Swar-chia-plot-manager
command promt phyton
GUI for farming

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
